src/style-spec/bin/gl-style-validate

```diff
@@ -39,4 +46,5 @@ function help() {
     console.log('');
     console.log('options:');
     console.log('--json  output errors as json');
+    console.log('--mapbox-api-supported validate compatibility with Mapbox Styles API');
```

Comment:
nit: the `--json` help has two spaces between the flag and the description. do the same here for consistency?

```suggestion
    console.log('--mapbox-api-supported  validate compatibility with Mapbox Styles API');
```